src/main/java/io/cdap/plugin/http/source/common/http/OAuthUtil.java

```diff
@@ -92,5 +94,29 @@ public static String getAccessTokenByServiceAccount(BaseHttpSourceConfig config)
     }
     return accessToken;
   }
+
+  public static String getAccessTokenByClientCredentials(CloseableHttpClient httpclient, String tokenUrl,
+                                                         String clientId, String clientSecret, String grantType)
+          throws IOException {
+    URI uri;
+    try {
+      uri = new URIBuilder(tokenUrl).setParameter("grant_type", grantType).build();
```

Comment:
define constants where ever any keyword getting used multiple times or has scope of getting used in future

Reply:
Constants are defined for both client credentials method and refresh token method.